src/components/Hero/Hero.tsx

```diff
@@ -239,19 +253,19 @@ export const Hero: ChakraComponent<
               },
             };
       } else if (variant === "tertiary") {
-        const tertiaryBgColor = backgroundColor
-          ? backgroundColor
-          : defaultBackgroundColor;
-        backgroundImageStyle = backgroundColor
-          ? { bgColor: backgroundColor }
-          : { bgColor: tertiaryBgColor };
+        backgroundImageStyle =
+          backgroundColor || textBackgroundColor
+            ? { bgColor: backgroundColor || textBackgroundColor }
+            : { bgColor: defaultBackgroundColor };
       }
 
       /** The "tertiary" variant doesn't have an apparent content box, so that
        * element will default to a transparent background in the "teriary"
        * variant. */
       contentBoxStyling = {
-        ...(foregroundColor && { color: foregroundColor }),
+        ...((foregroundColor || textColor) && {
+          color: foregroundColor || textColor,
```

Comment:
Minor, but switch these around to show in the code that we prefer one over the deprecated var.